### PR TITLE
Add MOBILE proposals

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

File: mobile-proposals.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "MOBILE Working Group (Even Class) Elections - Q4 2023",
+    "uri": "https://github.com/KeithRettig/HIP/blob/2023Q4_ModularVoting/2023Q4_ModularVoting__Summary",
```
Comment (abhay) [inline mobile-proposals.json:4]: ideally this is moved into this repo instead.

Comment (abhay) [inline mobile-proposals.json:4]:
```suggestion
    "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__00__Summary.md",
```

File: mobile-proposals.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      { "uri": null, "name": "Heated Lime" },
+      { "uri": null, "name": "Keith Rettig" },
+      { "uri": null, "name": "Max Gold" },
+      { "uri": null, "name": "Kennyy" },
+      { "uri": null, "name": "cheflifeworld" },
+      { "uri": null, "name": "Erfanjhan" },
+      { "uri": null, "name": "Unikt39" },
+      { "uri": null, "name": "Brainstormer" },
+      { "uri": null, "name": "KΞ7IN" },
+      { "uri": null, "name": "greenz" }
```
Comment (abhay) [inline mobile-proposals.json:17]:
```suggestion
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_heatedlime.md", "name": "Heated Lime" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_keithrettig.md", "name": "Keith Rettig" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_Capc0m.md", "name": "Max Gold" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_kennyy6399.md", "name": "Kennyy" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_cheflifeworld.md", "name": "cheflifeworld" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_Erfanjhan.md", "name": "Erfanjhan" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_Unikt39.md", "name": "Unikt39" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_Brainstormer.md", "name": "Brainstormer" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_kevin314.md", "name": "KΞ7IN" },
      { "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__CandidateMessage_greenzdewi.md", "name": "greenz" }
```

**Review by abhay — CHANGES_REQUESTED** [review]
Added changes from candidates and vote text.

File: mobile-proposals.json (hunk 2 above)
Comment (ChewingGlass) [reply mobile-proposals.json:17]: URIs don't do anything yet. Not sure if they will be markdown when they do, so not accepting this change.

File: mobile-proposals.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "MOBILE Working Group (Even Class) Elections - Q4 2023",
+    "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/ebbe80bc2d992e5d9c64530391523683c67c8c0e/2023Q4__MOBILE__00__Summary.md",
```
Comment (abhay) [inline mobile-proposals.json:4]:
```suggestion
    "uri": "https://gist.githubusercontent.com/hiptron/036867431eb2153a52babe30fb5dbdf3/raw/50f037e807ad844e87e0ff335ffe605bedb48d57/2023Q4__MOBILE__00__Summary.md",
```

**Review by abhay — APPROVED** [review]

**Pull request merged**